### ISSUE TITLE
Import chemkin library error

### ISSUE THREAD

**Issue by AH9316** [ReactionMechanismGenerator/RMG-Py#1594]
### Topic
General area which your question is related to.
 - [ ] Installation of RMG
 - [ ] Running an RMG job
 - [ ] Using RMG API
 - [ ] Arkane (formerly CanTherm)
 - [ ] Dependencies
 - [ ] An error message 

### Context
We try to import the library into RMG database. The RMG shows us that "NotImplementedError: Can't check isomorphism of lists with 4 species/molecules".
### Question
How to fix it?

**Comment by alongd** [comment]
My guess is that you probably have reactions with more than three reactants or products. Our solver does not support these, and looks like the parser was set not to process them.
If this is not the case, or if you're unsure, paste the complete error trace message you get, and upload the files you are converting

**Comment by AH9316** [comment]
Thank you for your response. We checked the reactions and we found that there is one reaction,which is "C7H16<=>CH3+2C2H4+C2H5", causes this error. We imported chemkin library successfully when we commented it out. In my opinion, I don't think the reason for this error is that we have more than 3 products in the right hand side of the reaction equation, because we successfully imported library with other reactions that have more than 3 products. 

Below are the chemkin file and species dictionary we create, hope it can help you guys to improve RMG import function and figure out why this error appears. 
[chemkin_speciesdictionary.zip](https://github.com/ReactionMechanismGenerator/RMG-Py/files/3144688/chemkin_speciesdictionary.zip)

**Comment by alongd** [comment]
Could you try replacing "C7H16<=>CH3+2C2H4+C2H5" with "C7H16<=>CH3+C2H4+C2H4+C2H5" just for the exercise? Perhaps the coefficient before `C2H4` was problematic, and if so maybe RMG should give a more helpful error message in the future

**Comment by AH9316** [comment]
It does not work and the RMG shows the same error:
"NotImplementedError: Can't check isomorphism of lists with 4 species/molecules
Exception NotImplementedError: NotImplementedError("Can't check isomorphism of lists with 4 species/molecules",) in 'rmgpy.reaction.isomorphic_species_lists' ignored
NotImplementedError: Can't check isomorphism of lists with 4 species/molecules
Exception NotImplementedError: NotImplementedError("Can't check isomorphism of lists with 4 species/molecules",) in 'rmgpy.reaction.isomorphic_species_lists' ignored"

**Comment by alongd** [comment]
OK, so seems like the error message is informative enough after all.

**Comment by AH9316** [comment]
As a user, personally, this error message is not very obviously. We actually spend a lot of time yesterday to find this reaction. When I first saw this message, I thought there is something wrong with my species dictionary. RMG is a very powerful software, we actually got good results using mechanisms that generated by RMG. And we still get some problems with some big molecular, such as benzene, toluene. I hope we can see the version 2.4.0 soon at the end of May.
